lib/components/atoms/Button/Button.js

```diff
@@ -51,6 +53,7 @@ Button.defaultProps = {
   secondary: false,
   tertiary: false,
   disabled: false,
+  transparent: false,
```

Comment:
I am thinking to change these boolean variations to named ones 

```Javascript
  primary: false,
  secondary: false,
  tertiary: false,
  disabled: false,
  transparent: false,
```

@ankurkakroo2 thoughts?